doc/nrf/device_guides/working_with_nrf/nrf70/features.rst

```diff
@@ -43,45 +40,95 @@ Devices in the nRF70 Series are supported by the following boards in the `Zephyr
        | :ref:`Getting started <ug_nrf52_gs>`
        | `User Guide <nRF52840 DK User Guide_>`_
-   * - nRF9160 DK
+   * - :ref:`zephyr:nrf9160dk_nrf9160`
      - nRF7002 EK
      - PCA10090
      - ``nrf9160dk_nrf9160_ns``
      - | `Product Specification <nRF9160 Product Specification_>`_
        | :ref:`Getting started <ug_nrf9160_gs>`
        | `User Guide <nRF9160 DK Hardware_>`_
-   * - Thingy:53
+   * - :ref:`zephyr:nrf9161dk_nrf9161`
+     - nRF7002 EK
+     - PCA10153
+     - ``nrf9161dk_nrf9161_ns``
+     - | `Product Specification <nRF9161 Product Specification_>`_
+       | `User Guide <nRF9161 DK Hardware_>`_
+   * - :ref:`zephyr:thingy53_nrf5340`
      - nRF7002 EB
      - PCA20053
      - ``thingy53_nrf5340_cpuapp``
      - | :ref:`Getting started <ug_thingy53_gs>`
        | `User Guide <Nordic Thingy:53 Hardware_>`_
 
+Wi-Fi shields
+==============
+
+The following Wi-Fi shields are available and defined in the :file:`nrf/boards/shields` folder.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Hardware platform
+     - PCA number
+     - Build target
+     - Documentation
+   * - nRF7002 :term:`Evaluation Kit (EK)`
+     - PCA63556
+     - ``nrf7002ek``
+     - | :ref:`Getting started <ug_nrf7002ek_gs>`
+       | `User Guide <nRF7002 EK User Guide_>`_
+   * - nRF7002 EK with emulated support for the nRF7001 IC
+     - PCA63556
+     - ``nrf7002ek_nrf7001``
+     - | :ref:`Getting started <ug_nrf7002ek_gs>`
+       | `User Guide <nRF7002 EK User Guide_>`_
+   * - nRF7002 EK with emulated support for the nRF7000 IC
+     - PCA63556
+     - ``nrf7002ek_nrf7000``
+     - | :ref:`Getting started <ug_nrf7002ek_gs>`
+       | `User Guide <nRF7002 EK User Guide_>`_
+   * - nRF7002 :term:`Expansion Board (EB)`
+     - PCA63561
+     - ``nrf7002eb``
+     - | :ref:`Getting started <ug_nrf7002eb_gs>`
+       | `User Guide <nRF7002 EB User Guide_>`_
+
+Supported protocols
+*******************
+
+The nRF70 Series devices support the Wi-Fi protocol.
+Wi-Fi is a half-duplex packet-based protocol operating on a fixed channel, using CSMA/CA for channel access.
+
+See the :ref:`ug_wifi` documentation for information related to the Wi-Fi protocol.
+
 Supported Wi-Fi standards and modes
-***********************************
+===================================
 
 The nRF70 Series wireless companion ICs add Wi-Fi 6 support to a host device that includes IP-based networking support.
 Wi-Fi 6 aligns with IEEE 802.11ax and all earlier versions of the IEEE 802.11 suite of wireless LAN standards.
 
-Currently, :term:`Station mode (STA)` and :term:`Software-enabled Access Point (SoftAP or SAP)` mode are supported.
+Currently, :ref:`Station (STA) <wifi_station_sample>`, :ref:`Software-enabled Access Point (SoftAP or SAP) <nRF70_soft_ap_mode>`, :ref:`Scan-only <ug_nrf70_developing_scan_operation>` and :ref:`TX injection <ug_nrf70_developing_raw_ieee_80211_packet_transmission>` modes are supported.
```

Comment:
Is there a reason to call this Scan-only ? Don't like it so much. How about "Wi-Fi Scan", or "SSID Scan", or simply "Scan"?

Also, how about using "Raw packet TX"?

Reply:
@bama-nordic @sachinthegreen ^^

Reply:
Changed to Scan mode.

TX injection mode is mentioned here in this document, https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/wifi/raw_tx_operation.html#tx-injection-mode.

Let me know if we decide to change, then I think it needs to be updated here as well.

Reply:
Yes, I think the TX Injection mode is sort of an "internal" technical naming, which is OK to have, but a more top-level, user-friendly name for this Wi-Fi mode is Raw 8021.11 packet Transmission, as the title of that section suggests.

Reply:
I don't believe you need to change anything in the RAW TX packet chapter

Reply:
Ok.

Reply:
I don't think we should list `Scan` and `Tx injection` (or `Raw 802.11 packet transmission`) should be listed as `modes`. These should probably be listed as `functionalities`.

We are currently treating only the following as `Wi-Fi modes`:

- Station (STA)
- Software-enabled Access Point (SoftAP or SAP)
- Monitor

Reply:
`scan-only` is a mode in the `nRF700x` driver: See https://github.com/nrfconnect/sdk-nrf/blob/main/drivers/wifi/nrf700x/Kconfig#L32

Reply:
How about the following:

Currently the nRF70 supports the following modes:

- Station (STA) : Operates as a wireless client device.
- Software-enabled Access Point (SoftAP or SAP) : Operates as a virtual access point device.
- Scan : Operates as a scan only device.
<Monitor mode can appear here once it is available>

The nRF70 also supports the following functionalities:

- Raw 802.11 packet transmission : Allows injection of raw 802.11 frames in any of the above modes.
<Promiscuous reception can appear here when it is available>

Reply:
The `Radio test` above can be linked to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-13827/nrf/samples/wifi/radio_test/README.html

Reply:
This is fine with me, thanks :)